src/backend/marsha/core/tests/test_xapi.py

```diff
@@ -91,6 +93,7 @@ def test_xapi_statement_enrich_statement(self):
                     },
                 },
                 "id": "uuid://68333c45-4b8c-4018-a195-5d5e1706b838",
+                "name": {"fr-FR": "test video xapi"},
```

Comment:
but the locale may be in a wrong format. You have methods in marsha or django to convert to the "full language"

Reply:
look here: https://github.com/openfun/marsha/blob/master/src/backend/marsha/core/utils/react_locales_utils.py#L4

Reply:
Even if the locale is set in the settings ?

Reply:
It's not set with this format:
https://github.com/openfun/marsha/blob/master/src/backend/marsha/settings.py#L179
and 
https://github.com/openfun/marsha/blob/master/src/backend/marsha/settings.py#L185

By the way LANGUAGE_CODE shoud be one of the choices of LANGUAGES imo.

Reply:
Ok I made a modification

Reply:
But now it's "fr_FR" instead of "fr-FR". Is it why xAPI wants?

Reply:
No it's not... It expects RFC 5646

Reply:
So in fact what we want it's realle what `LANGUAGE_CODE` contains.

Reply:
```
A language map is a dictionary where the key is a RFC 5646 Language Tag, and the value is a string in the language specified in the tag. This map SHOULD be populated as fully as possible based on the knowledge of the string in question in different languages.

The shortest valid language code for each language string is generally preferred. The ISO 639 language code plus an ISO 3166-1 country code allows for the designation of basic languages (e.g., es for Spanish) and regions (e.g., es-MX, the dialect of Spanish spoken in Mexico). If only the ISO 639 language code is known for certain, do not guess at the possible ISO 3166-1 country code. For example, if only the primary language is known (e.g., English) then use the top level language tag en, rather than en-US. If the specific regional variation is known, then use the full language code.
```

Reply:
So should it be:
```python
to_locale(settings.LANGUAGE_CODE).replace("_", "-") 
```

Reply:
Yes exactly !